packages/@okta/vuepress-site/docs/guides/enable-cors/main/index.md

```diff
@@ -13,57 +13,61 @@ This guide explains Cross-Origin Resource Sharing (CORS), why it’s useful, how
 * Grant cross-origin access to the Okta API from your web apps.
 * Test your Okta CORS configuration.
 
+#### What you need
+An [Okta Developer Edition org](https://developer.okta.com/signup/)
+
 ---
 
 ## About CORS
 
-[Cross-Origin Resource Sharing (CORS)](https://www.w3.org/TR/cors/) is a mechanism that allows a web page to make an AJAX call using [XMLHttpRequest (XHR)](https://xhr.spec.whatwg.org/). Use XHR to call a domain that is different than the domain where the script was loaded. Such cross-domain requests would otherwise be forbidden by web browsers as indicated by the [same origin security policy](https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy). CORS defines a [standardized](http://www.w3.org/TR/cors/) way in which the browser and the server can interact to determine whether to allow the cross-origin request.
+[Cross-Origin Resource Sharing (CORS)](https://www.w3.org/TR/cors/) is a mechanism that allows a web page to make an AJAX call using [XMLHttpRequest (XHR)](https://xhr.spec.whatwg.org/).
 
-In Okta, CORS allows JavaScript, which is hosted on your website, to make an XHR to the Okta API with a token. See [OAuth 2.0](/docs/guides/implement-oauth-for-okta/).
+You can use XHR to call a domain that is different from the domain where the script was loaded. Web browsers typically don't allow cross-domain requests because of the [same origin security policy](https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy). However, CORS defines a standardized way for the server and browser to interact. Then CORS determines whether to allow a cross-origin request.
```

Comment:
```suggestion
You can use XHR to call a domain that's different from the domain where the script was loaded. Web browsers typically don't allow cross-domain requests because of the [same origin security policy](https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy). However, CORS defines a standardized way for the server and browser to interact. Then CORS determines whether to allow a cross-origin request.
```